modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -3173,6 +3173,40 @@ definition {
 			value1 = "An unexpected error occurred while uploading your file.");
 	}
 
+	@description = "LPS-176823 - Verify that the Output Field on Formula Field is required"
+	@priority = 4
+	test OutputFieldIsRequiredOnFormulaField {
+		property portal.acceptance = "true";
+
+		task ("Given: an Object") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 176823",
+				objectName = "CustomObject176823",
+				pluralLabelName = "Custom Objects 176823");
+
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectSystemObject(label = "Custom Object 176823");
+
+			ObjectAdmin.goToFieldsTab();
+		}
+
+		task ("When: managing a Formula Field") {
+			ObjectAdmin.addObjectFieldViaUI(
+				fieldLabel = "Custom Formula",
+				fieldType = "Formula",
+				formulaFieldOption = "Decimal");
+		}
+
+		task ("Then:It's not possible to select mandatory option") {
```

Comment:
If you need to assert that the output field is a required let's change it to "Then: the output field is a mandatory field"